sfepy/version.py

```diff
@@ -36,7 +36,7 @@ def get_basic_info(version=__version__):
     for up_dir in ['..', '.']:
         top_dir = op.normpath(op.realpath(op.join(op.dirname(__file__),
                                                   up_dir)))
-        aux = op.join(top_dir, 'README.rst')
+        aux = op.join(top_dir, 'VERSION')
```

Comment:
Where does the `VERSION` come from? When I clone `rc/reorganize-scripts-entry-points` and run `python3 setup.py build_ext --inplace` `VERSION` does not exist and the top level directory can not be determined.

Reply:
Good catch, fixed.